mne/cov.py

```diff
@@ -1713,6 +1713,8 @@ def _estimate_rank_meeg_signals(data, info, scalings, tol=1e-4,
         thresholded to determine the rank are also returned.
     """
     picks_list = _picks_by_type(info)
+    if scalings is None:
+        scalings = dict(mag=1e15, grad=1e13, eeg=1e6)
```

Comment:
What's the issue here. I'm more and more convinced by the way that havin defaults in private functions, especially if they are deep inside a code path is not so good because it increases complexity.